solc_json_parser/parser.py

```diff
@@ -153,6 +153,22 @@ def symbols_to_ids_from_ast_v7(ast: Dict[Any, Any]) -> Dict[str, int]:
     return {k: v[0] for m in syms for k, v in m.items()}
 
 
+def get_increased_version(current_version: str) -> str:
+    # convert current version str to Version object
+    current_version_obj = Version(current_version)
+    versions = solcx.get_installable_solc_versions()
```

Comment:
`get_installable_solc_versions` will retrieve the list from network, this can be slow. Please cache the result

Reply:
Hi @YoungHS-NUS , I think we should cache `get_installable_solc_versions` instead of the `get_increased_version`